filetracker/servers/migration.py

```diff
@@ -19,11 +19,11 @@ def __init__(self, redirect_url, dir=None):
         self.redirect_url = redirect_url
 
     def handle_redirect(self, environ, start_response, present_handler):
-        path = os.path.join(self.dir, self._get_path(environ))
+        path = os.path.join(self.dir, self.get_path(environ))
         if os.path.isfile(path):
             return present_handler(environ, start_response)
 
-        new_url = self.redirect_url + '/' + self._get_path(environ)
+        new_url = self.redirect_url + environ['PATH_INFO']
```

Comment:
I guess PATH_INFO always starts with a slash, but what if it didn't? Maybe we could use some kind of URL concatenation function?